packages/web/src/components/alt-button/FavoriteButton.module.css

```diff
@@ -35,8 +35,8 @@
 }
 
 .heart {
-  height: 22px;
-  width: 22px;
+  height: 24px;
```

Comment:
can drop these and just make sure the icon has size='l'? which it has by default anyway so maybe can drop entirely?

Reply:
It's not using the Icon component so this can't be removed. Can probably be updated to use the proper component, but there is a weird icon map to multiple icons that I didn't want to mess with